packages/web-components/src/utilities/appearance-behavior.ts

```diff
@@ -1,15 +0,0 @@
-import { ElementStyles } from '@microsoft/fast-element';
-import { PropertyStyleSheetBehavior } from '@microsoft/fast-foundation';
-
-/**
- * Behavior that will conditionally apply a stylesheet based on the elements
- * appearance property
- *
- * @param value - The value of the appearance property
- * @param styles - The styles to be applied when condition matches
- *
- * @public
- */
-export function appearanceBehavior(value: string, styles: ElementStyles) {
-  return new PropertyStyleSheetBehavior('appearance', value, styles);
```

Comment:
Per Badge comment, deleted these to show the naive thinking and open convo around their purpose. It advantageous to remove code and concepts from the code, if possible.

Reply:
I'm fine removing these based on our convos around multiple stylesheets, etc - I'll note the purpose here:

We have the ability to create "behaviors" which can conditionally add styles. So a big use case here is something like "forced colors" for high contrast which can be conditional included. If it's not in forced-colors mode, those styles aren't downloaded and run unless the MQ hits. Based on the feedback we've gotten from profiling Atomic CSS, I think we start with monolithic stylesheets for _most_ things and remove this per your note here.

Reply:
One place we'd likely leverage this though would be w/ ElementInternals. We can ship the polyfill styles in a behavior which would be conditional on a `@supports` query. That way, we get great perf with low overhead in supported browsers and the additional styles get served when not supported. Just an example, but these aren't that :)